src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -473,6 +505,35 @@ def create_or_update_test_with_config(
         new_body["loadTestConfiguration"]["splitAllCSVs"] = yaml_test_body[
             "loadTestConfiguration"
         ]["splitAllCSVs"]
+
+    new_body["autoStopCriteria"] = {}
+    if autostop_criteria is not None:
+        new_body["autoStopCriteria"] = autostop_criteria
+    elif yaml_test_body.get("autoStopCriteria") is not None:
+        new_body["autoStopCriteria"] = yaml_test_body["autoStopCriteria"]
```

Comment:
do we need to convert the yaml's autostop criteria to request ?

Reply:
yes, we are converting that in the `convert_yaml_to_test` method